src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds

```diff
@@ -13,7 +13,7 @@
     </Project>
     <Project Condition="'$(TargetsWindows)' == 'true'" Include="win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
       <OSGroup>Windows_NT</OSGroup>
-      <Platform>amd64</Platform>
+      <Platform Condition="'$(Platform)' == ''">amd64</Platform>
```

Comment:
I don't think this is the right approach. The idea is that you would ad another entry for x86 directly. In general the .builds file is supposed to multiplex all the possible configurations.